research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -691,7 +695,46 @@ Each of these options is significantly different from the others. To properly ev
 
 That document discusses the pros and cons for each alternative. After exploring these options, the [HTML content attribute](#html-content-attribute) approach [was resolved by OpenUI](https://github.com/openui/open-ui/issues/455#issuecomment-1050172067) to be the best overall.
 
-## Design decisions
+## Why a content attribute?
+
+Again, refer to the [Other Alternatives Considered](/components/popup.proposal.alternatives) document for an exhaustive look at the other alternatives. That document runs through a fairly complete design of the other alternatives, to see what they would look like.
+
+This section simply tries to summarize the primary reason that a content attribute was chosen to enable the pop-up behavior: a content attribute allows *behavior* to be applied to *any element*. That is useful:
+
+```html
+<dialog popup=auto>
+  This is a "light dismiss" dialog
+  <button>Ok</button>
+  <button>Cancel</button>
+</dialog>
+```
+
+Here, the developer is building **a dialog**. Since HTML [strongly encourages](https://html.spec.whatwg.org/multipage/dom.html#semantics-2) the use of the correct element that properly represents the **semantics of the content**, the proper element in this case is a `<dialog>`.
+
+```html
+<menu popup=auto>
+  <li><button>Item 1</button></li>
+  <li><button>Item 2</button></li>
+  <li><button>Item 3</button></li>
+</menu>
+```
+
+Similarly here, the developer is building **a menu**, so they should use the `<menu>` element. This is natural and expected.
+
+In both cases, not only does the use of the semantically-correct element make it easier to understand the content, it also enforces the appropriate content rules and behaviors that are specific to each element. Further, using the correct element allows the UA to correctly represent the content in the accessibility tree, making assistive technologies better able to assist users in navigating the content.
+
+In an alternative proposal where the pop-up behavior is enabled via a special element, e.g. `<popup>`, all of the above would need to be carefully managed by the developer via ARIA roles and attributes:
+```html
+<popup role=dialog>
+<popup role=menu>
```

Comment:
correction here, because although one would think that `<menu>` == `role=menu`, the menu element is really just a semantically different way of specifying a `<ul>` elment.

```suggestion
<popup role=list>
```

Reply:
Ugh. Really? Ugh. Ok, done. 😢

Reply:
that is the correct response.